dfastmi/batch.py

```diff
@@ -28,14 +28,19 @@
 """
 
 from typing import Optional, List, Union, Dict, Any, Tuple, TextIO
-from dfastmi.RiversObject import RiversObject
+import sys
+from dfastmi.io.RiversObject import RiversObject
 from dfastmi.kernel.core import Vector, BoolVector, QRuns
```

Comment:
Not sure how this dfastmi.kernel.core import is functioning, might be something to tackle in batch refactor.
Not an change for this issue,.

Reply:
Will be picked up in [DFAST-103](https://issuetracker.deltares.nl/browse/DFAST-103)	
[Investigate Refactoring batch](https://issuetracker.deltares.nl/browse/DFAST-103)